### ISSUE TITLE
TLDR missions for all Mission tables

### ISSUE THREAD

**Issue by I-AM-T3X** [TLDRMissions/TLDRMissions#340]
TLDR version: One mission table addon to RULE them all 

Basically add the option or maybe a plugin to TLDR missions  that will basically send off missions for all tables, As a gold maker i have been loving your addon but i would really like to not have to use 3 addons to do missions on all my tables, from the garrison to order hall and war campaign, liek there is still some good gold to be made off these tables from older expacs and they never really get updates anymore so adding fuctionallity to your exisiting addon to be able to do them would be a god send for people that still use them to bring in gold and resources we use to to make gold, you wouldn't even have make it on by default, it could be a toggle in the setting that we could just turn on

**Comment by teelolws** [comment]
That would take a lot of time, for not much gain. It took me almost 3 months to make TLDR Missions. I released it in April but I was actually working on it for a lot longer before releasing it. There are already other addons that do this for those other mission tables. But I cannot just copy parts of their addons, as the Copyright belongs to them. I would have to start from scratch. It is just not something I am interested in investing a lot of time into.
